web/source/kmwkeyboards.ts

```diff
@@ -305,7 +305,7 @@ namespace com.keyman {
 
       // Fixed OSK font issue Github #7 (9/1/2015)
       if(typeof(lp['oskFont']) != 'undefined') {
-        sp['KOskFont'] = (typeof sp['KOskFont'] === 'undefined') ? new KeyboardFont(lp['oskfont'], fontPath) : sp['KOskFont'];
+        sp['KOskFont'] = (typeof sp['KOskFont'] === 'undefined') ? new KeyboardFont(lp['oskFont'], fontPath) : sp['KOskFont'];
```

Comment:
Does this mean OSK font has not been working properly since 2015?

Reply:
It's hard to say at this point.  It may have been broken and overlooked during the TypeScript refactor of this section, or it might not.